### PR TITLE
Fix cli overwrite option name

### LINKED ISSUE #14: Doesn’t overwrite
Check this out:

```sh
$ npm install cpy@3.3.0
$ mkdir -p source dest
$ echo Works! > source/a
$ echo Ooooops! > dest/a


$ cpy source/a dest
$ cat dest/a
» Ooooops!
```

### PR DESCRIPTION
**Pull request by floatdrop**
Closes #14

### CODE REVIEW

File: cli.js [hunk 1]
```diff
@@ -39,7 +39,7 @@ try {
 		cwd: cli.flags.cwd || process.cwd(),
 		rename: cli.flags.rename,
 		parents: cli.flags.parents,
-		overwrite: cli.flags.overwrite
+		overwrite: !cli.flags.noOverwrite
```
Comment (sindresorhus) [inline cli.js]:
```js
overwrite: cli.flags.overwrite !== false
```

minimist turns `--no-overwrite` into `overwrite === false`.

**Comment by sindresorhus** [comment]
That was fast! Appreciate it :)

File: cli.js (hunk 1 above)
Comment (floatdrop) [inline cli.js]: @sindresorhus didn't know that! But still `!cli.flags.noOverwrite` looks better than `cli.flags.overwrite === undefined ? true : cli.flags.overwrite` - should I change it?

Comment (schnittstabil) [inline cli.js]: @floatdrop: I think @sindresorhus  proposing:
```js
		parents: cli.flags.parents,
		overwrite: cli.flags.overwrite !== false
	}, errorHandler);
```

Comment (floatdrop) [inline cli.js]: :blush:

**Comment by sindresorhus** [comment]
:cocktail:

**Pull request merged**